src/config.cc

```diff
@@ -71,6 +71,7 @@ void MainConfig::retrieve_config() {
   Singleton::Config::window()->default_size = {cfg.get<int>("default_window_size.width"), cfg.get<int>("default_window_size.height")};
   Singleton::Config::terminal()->make_command=cfg.get<std::string>("project.make_command");
   Singleton::Config::terminal()->cmake_command=cfg.get<std::string>("project.cmake_command");
+  Singleton::Config::terminal()->clang_format_command=cfg.get<std::string>("project.clang_format_command", "clang-format");
   Singleton::Config::terminal()->history_size=cfg.get<int>("terminal_history_size");
```

Comment:
I have an idea solving this without duplication, i'll make a pull request ~tomorrow with a suggestion.